go-controller/pkg/kubevirt/types.go

```diff
@@ -2,4 +2,11 @@ package kubevirt
 
 const (
 	AllowPodBridgeNetworkLiveMigrationAnnotation = "kubevirt.io/allow-pod-bridge-network-live-migration"
+	SubnetSwitchNameLabel                        = "kubevirt.io/subnet-switch-name"
+	VMLabel                                      = "kubevirt.io/vm"
 )
+
+type IPConfig struct {
```

Comment:
IMO this struct's name is a bit puzzling.

I would expect different information on something called `IPConfig`.

Reply:
> IMO this struct's name is a bit puzzling.
> 
> I would expect different information on something called `IPConfig`.

What do you think of 
```golang
type OriginalNetworkConfig struct {                                             
    SwitchName       string                                                     
    OvnPodAnnotation string                                                     
}             
```

Reply:
hm, not sure. I fail to see how an OVN pod annotation fits in a network config. 

Can we take a step back and analyze this with an example ? What data would you set here ? Let's assume these names for the time being.

Reply:
This is the "sticky" information that get copied from previews virt-launcher to the new one after live migration to allow CNI and controllers to configure the same network configuration that information is the ovn pod annotation with addresses, gateways, etc... and the original switch owning the address subnet, needed to reserve the address in case of a restart for example.

Reply:
I see. Thanks. 

I would be totally fine with that name - `OriginalNetworkConfig` - if you'd unmarshal the annotation and set the default network config in it.

Reply:
Does `OVNNetworksStatus` make sense to you ? 
i.e.
```golang
type OriginalNetworkConfig struct {                                             
    SwitchName               string                                                     
    OVNNetworksStatus string // OVN networks annotation
}             
```

Reply:
> Does `OVNNetworksStatus` make sense to you ? i.e.
> 
> ```go
> type OriginalNetworkConfig struct {                                             
>     SwitchName        string                                                     
>     OVNNetworksStatus string // OVN networks annotation
> }             
> ```

Fine by me

Reply:
OK, I think this one is better:
```golang
type NetworkInfo struct {                                             
    OriginalSwitchName        string                                                     
    Status string // OVN networks annotation
}
```
the "original" thing that is relevant is actually the switch name.